### DISCUSSION TITLE
Group showing as Inactive even though imported as active

### DISCUSSION THREAD

**Discussion by OSAwebsite** [code4recovery/12-step-meeting-list#1316] [Q&A] [answered]
Hi,

I'm importing active groups to TSML, but they are showing as inactive on the website. Any suggestions?

**Comment by tim-rohrer** [comment]
Please elaborate on what you mean by "inactive".

Meetings added through a csv import need to meet the same basic requirements as meetings added through the Edit Meeting page. Specifically, they must  have an address that can be geocoded by Google. Data fields must match the specification; the quickest way to accomplish this is by using the provided csv template.

Beyond these suggestions, we'll need much more information before being able to suggest additional troubleshooting steps.

**Reply by OSAwebsite** [reply]
Hi,

So I import meetings to the website via FileMaker that was set up by our previous website coordinator. I haven't had any issues with this before now. 

I'm wondering if the problem is because the location is either temporarily closed and meeting on Zoom or they are electronic only and don't have their Zoom information listed?

Is there not a spot in TSML to just list the meeting as "active" or "inactive" directly from the website?

**Reply by tim-rohrer** [reply]
Can you tell me specifically what you mean by "showing as inactive?" I'm stumbling on this because if you mean the website shows the meeting as temporarily closed, then I suspect that code is being imported. However, if you're showing the meeting as not being published, then I'd be more inclined to look at addresses.

Please remember, **every** meeting must have an address, even if it is an approximate (i.e, city and state). If the address is complete (i.e., street, city and state) and the TC code isn't imported, then the meeting will show correctly (as long as the address geocodes).

**Reply by kiyote33** [reply, marked as answer]
Hi @OSAwebsite, I suspect you may have answered your own question here. As Tim mentioned, Online Only meetings must have an approximate address and the Online Details URL or Phone fields must be populated with valid information.  As you mentioned, if there is no zoom info in the URL field, then I would expect the meeting to show inactive. Hope this helps.

**Comment by OSAwebsite** [comment]
Hi,

The meetings have an approximate address - I have attached screenshots.

And how it is showing the meeting as inactive.
<img width="265" alt="Screenshot 2024-01-20 at 1 05 46 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/59859095/86c938c7-731a-45ff-8e2b-3f2ba3026991">
<img width="163" alt="Screenshot 2024-01-20 at 1 05 40 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/59859095/d344df32-c530-4e77-bdea-95edcd7176c4">
<img width="783" alt="Screenshot 2024-01-20 at 1 05 05 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/59859095/c36cd13d-1fa9-49e3-9645-51046b458639">

**Reply by OSAwebsite** [reply]
And TSML has accepted this address for other electronic meetings with no issues

**Reply by joshreisner** [reply]
that address looks good! what does it say under online meeting details?

<img width="842" alt="Screenshot 2024-01-20 at 10 08 37 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/89e5c8fa-1c9f-4743-854a-c08196c01aa4">
